docs/reference_guides/apps/grid_integration/multiperiod/Price_Taker.rst

```diff
@@ -0,0 +1,55 @@
+Price Taker
+===========
+
+Price takers are companies or individuals who must accept market prices since they lack the market share
+to directly influence the market price. Likewise, it is assumed that a price taker's resource or energy
+system is small enough such that it does not significantly impact the market. When coupled with multi-period modeling,
+the price-taker model is able to synthesize grid-centric modeling with steady-state process-centric modeling, as
+depicted in figure below.
+
+.. |pricetaker| image:: images/pricetaker.png
+  :width: 1200
+  :alt: Alternative text
+  :align: middle
+
+|pricetaker|
+
+The following equations represent the multi-period price taker model, where :math:`d` are design decisions,
+:math:`u` are operating decisions, :math:`δ` are power decisions, :math:`s` are scenarios (timeseries/representative days),
+:math:`w` is weight/frequency, :math:`R` is revenue, :math:`π` is price data,
+:math:`C` is capital and operating costs, :math:`g` is the process model, and :math:`h` is the temporal constraint.
+
+    .. math::
+
+       max_{d,u, δ} = \sum_{s ∈ S} \sum_{t ∈ T} w_{s}[R(d,u_{s,t},δ_{s,t},π_{s,t}) - C(d,u_{s,t},δ_{s,t})]
+
+    .. math::
+
+       g(d,u_{s,t},δ_{s,t}) = 0; ∀_{s} ∈ S, t ∈ T
+
+    .. math::
+
+       h(d,u_{s,t},δ_{s,t},u_{s,t+1},δ_{s,t+1}) = 0; ∀_{s} ∈ S, t ∈ T
+
+
+The price taker multi-period modeling workflow involves the integration of multiple software platforms into the IDAES optimization model
+and can be broken down into two distinct functions, as shown in the figure below. In part 1, simulated or historical
+ISO (International Organization for Standardization) data is used to generate locational marginal price (LMP)
```

Comment:
I am fairly that ISO stands for Independent System Operator or something similar.

Reply:
That's correct- "Independent System Operator".